scripts/score-project-quality-ai.py

```
@@ -355,6 +391,7 @@ def work(b):
 
 
 def judge_uniques_sync(client, uniques, model, batch_size, cache=None, cache_path=None):
+    set_usage_phase('recipient')
     """支出先（軸A/B）の判定。"""
```

Ruff (0.15.21)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)